Eatery/Views/Eateries/CollegetownMenuHeaderView.swift

```diff
@@ -0,0 +1,366 @@
+        distanceLabel.textColor = .gray
+        addSubview(distanceLabel)
+        
+        setupConstraints()
+    }
+    
+    func setupConstraints(){
+        backgroundImageView.snp.makeConstraints { make in
+            make.top.leading.trailing.equalToSuperview()
+            make.height.equalTo(258)
+        }
+        
+        paymentView.snp.makeConstraints { make in
+            make.trailing.equalToSuperview().inset(16)
+            make.centerY.equalTo(titleLabel.snp.centerY)
+        }
+        
+        titleLabel.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(16)
+            make.trailing.lessThanOrEqualTo(paymentView.snp.leading)
+            make.bottom.equalTo(backgroundImageView).inset(15)
+        }
+        
+        informationView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.top.equalTo(backgroundImageView.snp.bottom)
+            make.height.equalTo(105)
+        }
+        
+        statusLabel.snp.makeConstraints { make in
+            make.top.equalTo(informationView).inset(11)
+            make.leading.equalTo(informationView).inset(12)
+            make.trailing.lessThanOrEqualTo(informationView)
+            make.height.equalTo(19)
+        }
+        
+        hourLabel.snp.makeConstraints { make in
+            make.centerY.equalTo(statusLabel)
+            make.leading.equalTo(statusLabel.snp.trailing).offset(4.5)
+            make.trailing.lessThanOrEqualTo(informationView)
+            make.height.equalTo(17)
+        }
+        
+        cuisineLabel.snp.makeConstraints { make in
+            make.top.equalTo(statusLabel.snp.bottom).offset(2)
+            make.leading.equalTo(statusLabel)
+            make.height.equalTo(45)
+            make.width.equalTo(260)
+        }
+        
+        locationLabel.snp.makeConstraints { make in
+            make.top.equalTo(cuisineLabel.snp.bottom).offset(6)
+            make.leading.equalTo(cuisineLabel)
+            make.trailing.lessThanOrEqualTo(informationView)
+            make.height.equalTo(17)
+        }
+        
+        ratingView.snp.makeConstraints { make in
+            make.centerY.equalTo(statusLabel)
+            make.trailing.equalTo(informationView).inset(12)
+        }
+        
+        priceLabel.snp.makeConstraints { make in
+            make.top.equalTo(ratingView.snp.bottom).offset(13)
+            make.trailing.equalToSuperview().inset(12.5)
+            make.height.equalTo(19)
+            make.width.equalTo(30.5)
+        }
+        
+        distanceLabel.snp.makeConstraints { make in
+            make.centerY.equalTo(locationLabel)
+            make.trailing.equalToSuperview().inset(12.5)
+            make.height.equalTo(17)
+            make.width.lessThanOrEqualToSuperview()
+        }
+        
+    }
+    
+    func set(eatery: CollegetownEatery, userLocation: CLLocation?) {
```

Comment:
```suggestion
    public func set(eatery: CollegetownEatery, userLocation: CLLocation?) {
```

Public since used by CollegetownViewController

Reply:
Rather than public, we should leave the function without an access modifier to make the signature less cluttered.